apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/components/AssessmentQuestionTable.tsx

```diff
@@ -1,8 +1,10 @@
 import { useQuery, useQueryClient } from '@tanstack/react-query';
 import {
+  type ColumnFiltersState,
   type ColumnPinningState,
```

Comment:
The View menu looks weird because the "freeze column" button is on the final frozen column, not the first unfrozen column. And there is no button to unfreeze any column (first image below). If I click the "freeze column" button then it goes into a weird state (second image below).

<img width="188" height="644" alt="Image" src="https://github.com/user-attachments/assets/499e1db7-c8f0-40f5-87de-3d16c47d4a11" />

<img width="182" height="170" alt="Image" src="https://github.com/user-attachments/assets/d5beae3d-9797-4dd9-a8dd-d69845b34bb8" />

Reply:
Fixed!